frontend/packages/console-app/src/components/quick-starts/QuickStartDrawer.tsx

```diff
@@ -60,25 +63,32 @@ const QuickStartDrawer: React.FC<QuickStartDrawerProps> = ({
     return onClose();
   };
 
+  const headerStyles = classNames('co-quick-start-drawer-head', {
+    'pf-u-box-shadow-md-bottom':
```

Comment:
Probably best to used `sm` here. Better aligned with UX mocks.
```suggestion
    'pf-u-box-shadow-sm-bottom':
```

Reply:
Updated.